components/blocks/FilesToCards.vue

```diff
@@ -62,89 +61,102 @@
+            :allow-empty="true"
+          >
+          </multiselect>
+          <button class="ml-1 button is-normal is-warning" @click="clearAll">
+            Clear Filters
+          </button>
         </div>
         <div class="mb-1 is-flex">
-            <multiselect
-              v-model="sortBy"
-              :options="sortByOptions"
-              :close-on-select="true"
-              :clear-on-select="false"
-              :multiple="false"
-              :preselect-first="true"
-              placeholder="Sort by"
-              label="name"
-              track-by="name"
-              >\
-            </multiselect>
-            <button
-              class="ml-1 button is-normal is-warning"
-              @click="ascending = !ascending"
-            >
-              <i v-if="ascending" class="mdi mdi-sort-ascending"></i>
-              <i v-else class="mdi mdi-sort-descending"></i>
-            </button>
+          <multiselect
+            v-model="sortBy"
+            :options="sortByOptions"
+            :close-on-select="true"
+            :clear-on-select="false"
+            :multiple="false"
+            :preselect-first="true"
+            placeholder="Sort by"
+            label="name"
+            track-by="name"
+            >\
+          </multiselect>
+          <button
+            class="ml-1 button is-normal is-warning"
+            @click="ascending = !ascending"
+          >
+            <i v-if="ascending" class="mdi mdi-sort-ascending"></i>
+            <i v-else class="mdi mdi-sort-descending"></i>
+          </button>
         </div>
       </div>
       <div
         class="
           card-container
           is-flex
           mt-6
-          is-justify-content-space-evenly 
-          is-flex-wrap-wrap
+          is-justify-content-space-evenly is-flex-wrap-wrap
         "
       >
         <DCard
           v-for="(item, i) in sortedArray"
           :key="'help-card-' + i"
           class="mx-3 my-3 px-3"
           variant="light"
-          accent="link"
+          accent="warning"
           width="medium"
         >
           <template #header>
             <span v-if="item.group" class="radius-0 tag is-link has-text-light"
-              ><span class="icon"><i class="mdi mdi-account-multiple" /></span
               ><abbr :title="item.group | expandAcronym">
                 {{ item.group }}
               </abbr></span
             >
-            <h2 class="card-title">{{item.title}}</h2>
+            <h2 class="title has-text-black pt-3">{{ item.title }}</h2>
             <div v-if="item.date">Updated: {{ item.date }}</div>
-            <span v-if="item.authors" class="small has-text-link"
-              >Authors: {{ item.authors.map((a) => a.name).join(', ') }}</span
+            <span v-if="item.authors" class="subtitle has-text-black"
+              ><i class="mdi mdi-account-multiple p-1 m-1"></i
+              >{{ item.authors.map((a) => a.name).join(', ') }}</span
```

Comment:
is there also a github link already? or is that being added? if so, should render conditionally as a link if available

Reply:
@mcmcgrath13 a github link to the authors?

Reply:
yup! I think it gives username, but that's easily turned into a link